sshnp_docker_demo/README.md

```diff
@@ -0,0 +1,262 @@
+# sshnp_docker_demo
+
+This is a demo of sshing into one docker container from another docker container without having ports open (not even port 22). There are three docker containers: `sshnp`, `sshnpd`, and `sshrvd`. This demo involves using [sshnp](https://github.com/atsign-foundation/sshnoports).
+
+## Getting Started
+
+Follow these steps to get sshnp working on your machine using Docker.
+
+### 1. Prerequisites
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop/) and open it in the background. Check that you have docker installed. Run this in your terminal. Your output should be similar:
+
+```sh
+docker --version
+Docker version 23.0.5, build bc4487a
+```
+
+2. You will need 3 atSigns and their associated `.atKeys` files (one for each docker container). For each of your atSigns, put the `.atKeys` file into the `keys/` directory. For example, your file structure should be similar to:`sshnp/keys/@sshnp_key.atKeys`, `sshnpd/keys/@sshnpd_key.atKeys`, `sshrvd/keys/@sshrvd_key.atKeys`.
```

Comment:
It would be helpful to link to instructions on how to get an atSign, and how to get the .atKeys file for it.

Reply:
Done in https://github.com/atsign-foundation/at_demos/pull/97/commits/9ba4b06c6387eabfa82d03a47f353b2d99984433

Thanks!

Reply:
That video talks a LOT about PicoW. A more generic (text) guide would be better. If we don't have one then let's open a ticket to get one and have a TODO here.

Reply:
I'll find some time to make a new one today

Reply:
written instructions here - https://github.com/atsign-foundation/at_demos/pull/97/commits/752f9689f2175b9c26d49c801db0f6e520cc623c

added a todo for the video coming out soon